include/ks_qsmear.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <quda_internal.h>
+#include <quda.h>
```

Comment:
Both the `quda.h` and `quda_internal.h` includes should be needed in this include.  We should be striving in general to only include what we use.

Reply:
removed `quda.h`